src/markdown-pages/add-ons/weave.md

```diff
@@ -18,6 +18,8 @@ As of March 27, 2023, the Weave add-on is deprecated. The Weave add-on might be
 
 Weave Net creates a virtual network that connects containers across multiple hosts and enables their automatic discovery. With Weave Net, portable microservices-based applications consisting of multiple containers can run anywhere: on one host, multiple hosts or even across cloud providers and data centers.
 
+_**Warning**_: The use of symbolic links in conjunction with the Weave Addon could potentially cause inconsistencies, potentially hindering Weave's ability to establish reliable connections. This might result in operational errors or malfunctions. To minimize network connectivity issues, it is generally recommended to avoid symbolic links in Weave's network-related configurations.
```

Comment:
```suggestion
_**Warning**_: The use of symbolic links in conjunction with the Weave Addon could potentially cause inconsistencies, leading to disruptions in network connectivity. To minimize network connectivity issues, it is generally recommended to avoid symbolic links in Weave's network-related configurations.
```